packages/neon/neon_talk/lib/src/blocs/talk.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:meta/meta.dart';
+import 'package:neon_framework/blocs.dart';
+import 'package:neon_framework/models.dart';
+
+/// Bloc for fetching Talk rooms
+sealed class TalkBloc implements InteractiveBloc {
+  /// Creates a new Talk Bloc instance.
+  @internal
```

Comment:
why is the factory internal and not the entire class?

Reply:
This forces us to document the class as adding the internal annotation would no longer require docs.

Reply:
That's how it is done in all the other blocs as well.